content/appleboy/drone-ssh/index.md

```diff
@@ -82,66 +85,97 @@ pipeline:
     username: root
     password: 1234
     port: 22
-+   command_timeout: 10
     script:
       - echo hello
       - echo world
++   when:
++     branch: master
 ```
 
-Example configuration for execute commands on a remote server using ｀SSHProxyCommand｀:
+Example configuration for `tag` event:
 
 ```diff
 pipeline:
   ssh:
     image: appleboy/drone-ssh
     host: foo.com
     username: root
+    password: 1234
     port: 22
-    key: ${DEPLOY_KEY}
     script:
       - echo hello
       - echo world
-+   proxy_host: 10.130.33.145
-+   proxy_user: ubuntu
-+   proxy_port: 22
-+   proxy_key: ${PROXY_KEY}
++   when:
++     event: tag
 ```
 
-Example configuration for success build:
+Example configuration using password from secrets:
 
 ```diff
 pipeline:
   ssh:
     image: appleboy/drone-ssh
     host: foo.com
     username: root
-    password: 1234
+-   password: 1234
     port: 22
++   secrets: [ ssh_password ]
     script:
       - echo hello
       - echo world
-+   when:
-+     status: success
 ```
 
-Example configuration for tag event:
+Example configuration using ssh key from secrets:
 
 ```diff
 pipeline:
   ssh:
     image: appleboy/drone-ssh
     host: foo.com
     username: root
-    password: 1234
     port: 22
++   secrets: [ ssh_key ]
     script:
       - echo hello
       - echo world
-+   when:
-+     status: success
-+     event: tag
 ```
 
+Example configuration for exporting custom secrets:
+
+```diff
+pipeline:
+  ssh:
+    image: appleboy/drone-ssh
+    host: foo.com
+    username: root
+    password: 1234
+    port: 22
++   secrets: [ aws_access_key_id ]
++   envs: [ aws_access_key_id ]
+    script:
+      - export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
```

Comment:
It is already en anv variable, so why this additional export?

Reply:
See: 

Issue: https://github.com/appleboy/drone-ssh/issues/89
PR: https://github.com/appleboy/drone-ssh/pull/91